### PR TITLE
Bookstore add reducers and actions

### PR DESCRIPTION
**Pull request by omar25ahmed**
# Learning objectives:
- Understand the concept of state management.
- Use store, actions and reducers in React.
- Connect React and Redux.
- Connect an existing API via Redux and use the data to display something in a UI.

### CODE REVIEW

File: src/redux/books/Books.js [hunk 1]
```diff
@@ -9,19 +11,62 @@ const reducer = (state = initialState, action) => {
...
 };
 
-export const removeBook = (id) => ({
-  type: REMOVE_BOOK,
-  id,
-});
+// export const removeBook = (id) => ({
+//   type: REMOVE_BOOK,
+//   id,
+// });
```
Comment (micheaol) [inline src/redux/books/Books.js]: Please ensure that all commented codes are removed from your codebase. Commented code makes your code dirty.

File: src/redux/books/Books.js [hunk 2]
```diff
@@ -9,19 +11,62 @@ const reducer = (state = initialState, action) => {
...
 
-export const addBook = (payload) => ({
-  type: ADD_BOOK,
-  payload,
-});
+// export const addBook = (payload) => ({
+//   type: ADD_BOOK,
+//   payload,
+// });
```
Comment (micheaol) [inline src/redux/books/Books.js]: Please ensure that all commented codes are removed from your codebase. Commented code makes your code dirty.

**Review by micheaol — CHANGES_REQUESTED** [review]
Hi @omar25ahmed , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## TO HIGHLIGHT: 
- Your project is professional 💯 
- Your repo is professional 💯 
- Well structured files 💯 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by rotimiazeez — APPROVED** [review]
# Hi @omar25ahmed 😉, 

### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**